src/charm.py

```diff
@@ -257,8 +257,14 @@ def _alerting_config(self):
         if len(self.stored.alertmanagers) > 1:
             logger.warning('More than one altermanager found. Using first!')
 
-        manager = list(self.stored.alertmanagers.keys())[0]
-        alerting_config = self.stored.alertmanagers.get(manager, '')
+        targets = []
+        for manager in self.stored.alertmanagers.keys():
+            ip_addr = self.stored.alertmanagers[manager]['ip_addr']
+            port = self.stored.alertmanagers[manager]['port']
+            targets.append(f'{ip_addr}:{port}')
```

Comment:
We shouldn't use f-strings since they aren't supported in Python 3.5 and charms need to support 3.5 as far as I'm aware.

(I know this because I did the same thing and Chipaca pointed it out to me).

Reply:
@chipaca Can you verify this? In what cases are charms running on python 3.5?

Reply:
anything based on ubuntu 16.04 or centos 7.

Reply:
@chipaca yes but given that this is a k8s charm, is juju ever going to deploy the operator pod as one of those OSes?